src/battle_script_commands.c

```diff
@@ -6667,6 +6665,23 @@ static void HandleTerrainMove(u32 moveEffect)
     }
 }
 
+bool32 DoesAbilityBypassTypeImmunity(u8 battlerAttacker, u8 battlerTarget, u8 status)
```

Comment:
Why not just `CanPoisonType`? Without the status argument

Reply:
Done.